src/Elliptic/Systems/Poisson/Observe.hpp

```diff
@@ -0,0 +1,124 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Domain/Tags.hpp"
+#include "Elliptic/Systems/Poisson/Tags.hpp"
+#include "IO/Observer/Actions.hpp"
+#include "IO/Observer/ArrayComponentId.hpp"
+#include "IO/Observer/ObservationId.hpp"
+#include "IO/Observer/ObserverComponent.hpp"
+#include "IO/Observer/ReductionActions.hpp"
+#include "IO/Observer/VolumeActions.hpp"
+#include "NumericalAlgorithms/LinearOperators/DefiniteIntegral.hpp"
+#include "NumericalAlgorithms/LinearSolver/Tags.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Tags.hpp"
+#include "Utilities/Functional.hpp"
+#include "Utilities/MakeString.hpp"
+#include "Utilities/Numeric.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace Poisson {
+namespace Actions {
```

Comment:
For comparison, ScalarWave appears in the namespaces documentation with Actions and Solutions. I only see Solutions for Poisson, should Actions be added as well?

Reply:
It should be, but not in this file. Same with `Evolution/GrMhd/ValenciaDivClean`. The fixup commit below proposes to move this into an `Actions` subdirectory with a file `Actions.hpp` that documents the namespace. Is this an approach we want to follow?